### PR TITLE
API documentation common Attach* features

### PR DESCRIPTION
**Pull request by anindex**
Signed-off-by: anindex <an.thai.le97@gmail.com>

From issue #9

### CODE REVIEW

File: include/ignition/physics/BoxShape.hh [hunk 1]
```diff
@@ -80,6 +80,9 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature rigidly attachs a BoxShape to the specified
+    /// Link.
```
Comment (claireyywang) [inline include/ignition/physics/BoxShape.hh:85]: Maybe we can elaborate on "rigidly attach a BoxShape" here, something like "This feature constructs a new box shape, and attaches the pose of the box shape relative to the link frame."

File: include/ignition/physics/CylinderShape.hh [hunk 2]
```diff
@@ -92,6 +92,9 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature rigidly attachs a CylinderShape to the specified
```
Comment (claireyywang) [inline include/ignition/physics/CylinderShape.hh:96]: same as above

File: include/ignition/physics/PlaneShape.hh [hunk 3]
```diff
@@ -115,6 +115,8 @@ namespace physics
   };
 
   /////////////////////////////////////////////////
+  /// \brief This feature rigidly attachs a PlaneShape to the specified
```
Comment (claireyywang) [inline include/ignition/physics/PlaneShape.hh:118]: same

File: include/ignition/physics/PlaneShape.hh [hunk 4]
```diff
@@ -132,8 +134,8 @@ namespace physics
       ///   Name to give to the PlaneShape
       /// \param[in] _normal
       ///   Normal vector for the plane
-      /// \param[in] _offset
-      ///   Offset of the plane
+      /// \param[in] _point
+      ///   The point that the plane passes through (hence defining the plane)
```
Comment (claireyywang) [inline include/ignition/physics/PlaneShape.hh:138]: Is this point default to the origin `Point::Zero`?

**Review by claireyywang — CHANGES_REQUESTED** [review]
Some definitions need more clarification, but otherwise look good to me

File: include/ignition/physics/PlaneShape.hh (hunk 4 above)
Comment (anindex) [reply include/ignition/physics/PlaneShape.hh:139]: Yea, I think the plane will pass through point zero as default.

Comment (claireyywang) [reply include/ignition/physics/PlaneShape.hh:139]:
```suggestion
      ///   The point that the plane passes through (the origin by default)
```

**Review by claireyywang — COMMENTED** [review]
LGTM with a tad bit more detail

**Review by claireyywang — APPROVED** [review]

**Pull request merged**